Bottom: df808a62965028cd83032785ebc6f0f442b313cf
Top:    2135394d07ba2919cdce6c8340af54ffdc5a5ee6
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-02-15 15:50:17 +0800

Refresh of virito-net-keep-old

---

diff --git a/drivers/net/virtio_net.c b/drivers/net/virtio_net.c
index 1fffc8a0..3eb732f 100644
--- a/drivers/net/virtio_net.c
+++ b/drivers/net/virtio_net.c
@@ -1848,7 +1848,6 @@ static int virtnet_xdp_set(struct net_device *dev, struct bpf_prog *prog)
 	/* Should not happen, after reset, all queue pairs were
 	 * enabled by defualt.
 	 */
-	vi->xdp_queue_pairs = 0;
 	vi->curr_queue_pairs = vi->max_queue_pairs;
 	netif_set_real_num_tx_queues(dev, vi->curr_queue_pairs);
 	netif_set_real_num_rx_queues(dev, vi->curr_queue_pairs);
